### DISCUSSION TITLE
Grinding for Sonic World Next

### DISCUSSION THREAD

**Discussion by KrabbyCurse** [Techokami/SonicWorldsNext#71] [Ideas]
Like Sonic Superstars, I would like to see Grinding for Sonic World Next
![Adventure_2_Sonic_Grinding](https://github.com/Techokami/SonicWorldsNext/assets/141451277/584dc884-81fd-486b-ad3e-643f31f038e8)

**Comment by DimensionWarped** [comment]
We don't really have any animation for it. If you can add frames for Sonic, Knuckles, Tails, and Amy for grinding that fit within our quality standards I'll make a point of seeing what I can do.

**Reply by The-HandyDandyman** [reply]
Hey there! I saw this thread and decided for the fun of it to try to make grinding animations based on the poses used in Superstars (and little bit on sonic cd's speed launcher anim). These are moreso edits since I wanted to fit with what sprites already exist and I don't know if this is helpful at all, but I tried to give this a shot and I hope these can help out a bit: to anybody reading feel free to make modifications to these sprites as well. At the moment I've only done Sonic's and I've done a simple 2 frame loop, I'll continue posting more here as they get finished.
![sonic grind](https://github.com/Techokami/SonicWorldsNext/assets/134161546/52185fa1-6887-4396-bfd1-bed0565ee06f)
![sonic grind](https://github.com/Techokami/SonicWorldsNext/assets/134161546/85a46b15-c04d-447d-bf8f-6710ebef6210)
